src/index.tsx

```diff
@@ -158,7 +159,7 @@ export class SignatureCanvas extends Component<SignatureCanvasProps> {
   }
 
   toDataURL: SignaturePad['toDataURL'] = (type, encoderOptions) => {
-    return this.getSignaturePad().toDataURL(type, encoderOptions)
+    return this.getSignaturePad().toDataURL(type as string, encoderOptions as number)
```

Comment:
why was this type-cast necessary? we should have the exact same types and not force any subset